### ISSUE TITLE
mask

### ISSUE THREAD

**Issue by zhouhao028** [JCruan519/VM-UNet#66]
Hello, thank you very much for your work! I now have a question about why both train and val in dataset.py need mask input. Inputting the image and its mask in the training set helps the model recognize the area it needs to recognize, but why does the validation set also need mask input? Thank you very much for your work, thank you very much!
